exonum-java-binding/core/rust/ejb-app/TUTORIAL.md

```diff
@@ -18,90 +19,81 @@ in the [user guide](https://exonum.com/doc/version/0.10/get-started/java-binding
 
 ### Step 1. Configure Environment
 
-`EJB_ROOT` used in examples of this section corresponds to the Java Binding root directory.
-
-#### `LD_LIBRARY_PATH` and `EJB_LIBPATH`
+#### `LD_LIBRARY_PATH`
 
 `LD_LIBRARY_PATH` is required to locate native libraries used by Java Binding.
-You need to provide paths to:
-  - JVM library (e.g., `libjvm.so` on Linux).
-  - Rust standard library that is used to build the application.
-  - Application libraries used by Java Binding.
+You need to provide path to JVM library (e.g. `libjvm.so` on Linux).
 
 You can use the following script for this purpose:
 
 ```bash
 JAVA_HOME="${JAVA_HOME:-$(java -XshowSettings:properties -version 2>&1 > /dev/null | grep 'java.home' | awk '{print $3}')}"
 LIBJVM_PATH="$(find ${JAVA_HOME} -type f -name libjvm.* | xargs -n1 dirname)"
 
-RUST_LIB_PATH="$(rustup run 1.32.0 rustc --print sysroot)/lib"
-
-export EJB_LIBPATH="${EJB_ROOT}/core/rust/target/debug"
-
-export LD_LIBRARY_PATH="${LIBJVM_PATH}:${RUST_LIB_PATH}:${EJB_LIBPATH}"
+export LD_LIBRARY_PATH="${LIBJVM_PATH}"
 ```
 
 #### CLASSPATH
-Classpath is used to locate Java classes of your service and its dependencies, including 
-the Exonum Java Binding.
+Classpath is used to locate Java classes of your service and its dependencies.
 
 You may package your service in an Uber JAR using 
 the [Maven Shade Plugin](https://maven.apache.org/plugins/maven-shade-plugin/index.html)
-and pass a path to the service artefact during application configuration as `--ejb-classpath`
-parameter. Alternatively, you may assemble a classpath that includes the service and all of 
+and pass a path to the service artefact during application configuration as `--ejb-service-classpath`
+parameter. Alternatively, you may assemble a classpath that includes the path to service and all of 
 its dependencies and pass it instead.
 
+The service must use `provided` scope for all Exonum dependencies because they are included in the application.
+
 ### Step 2. Generate Node Configuration
 
 EJB App configuration is pretty similar to configuration of any other Exonum service,
 with a few additional parameters.
 
-**Note:** using `cargo run` command requires working from the `ejb-app` directory.
-This would be fixed in the future EJB versions.
-
 #### Generate Template Config
+Use `--ejb-module-name` for fully qualified name of your service module.
```

Comment:
Should it be a full Java class name?

Reply:
Isn't "fully qualified name" clear enough? Or are you saying that it might be unclear that service module is a Java class?

Reply:
Anyway, this documentation won't even survive till the next release because it is configured differently in "multiple-services"